.github/workflows/verify.yml

```diff
@@ -57,7 +57,7 @@ jobs:
   Publish:
     needs: [Build, Test]
     runs-on: ubuntu-latest
-    if: github.event.pull_request.merged == true && github.repository_owner == 'Apiman'
+    if: github.ref == 'refs/heads/main' && github.repository_owner == 'Apiman'
```

Comment:
And this works with this change?

Reply:
I hope so, this is now the same code as in the apiman `verify.yml`

Reply:
Ok 👍